src/macros.jl

```diff
@@ -849,51 +849,15 @@ esc_nonconstant(x::Number) = x
 esc_nonconstant(x::Expr) = isexpr(x,:quote) ? x : esc(x)
 esc_nonconstant(x) = esc(x)
 
-mutable struct VariableInfo{S, T, U, V}
-    haslb::Bool
-    lowerbound::S
-    hasub::Bool
-    upperbound::T
-    hasfix::Bool
-    fixedvalue::U
-    hasstart::Bool
-    start::V
-    binary::Bool
-    integer::Bool
-    name::String
-end
-
-# Returns the type of what `constructvariable!` would return with these starting positional arguments.
+# Returns the type of what `buildvariable` would return with these starting positional arguments.
```

Comment:
Does this still make sense? The positional arguments were replaced by the `VariableInfo` struct.